managed/src/main/java/com/yugabyte/yw/models/TaskInfo.java

```diff
@@ -91,7 +93,7 @@
   @Constraints.Required
   @Column(columnDefinition = "TEXT default '{}'", nullable = false)
   @DbJson
-  private JsonNode details;
+  private Map<String, Object>  details;
```

Comment:
This is not a Map. I followed the trail and this is of type ITaskParams.
Let me know if you have questions.

Reply:
Customers need this API so prioritize this and address HelthCheck separately.

Reply:
So what you really need is so that ITaskParam subtype can be deserialized back correctly:
https://stackoverflow.com/questions/30362446/deserialize-json-with-jackson-into-polymorphic-types-a-complete-example-is-giv

Thats too much to do and I do not think we need to expose the task params anyway.
Leave it as is. i.e. revert this part.